packages/web-components/src/components/va-table/test/va-table.e2e.ts

```diff
@@ -0,0 +1,116 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { axeCheck } from '../../../testing/test-helpers';
+
+describe('va-table', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-table></va-table>');
+
+    const element = await page.find('va-table');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('adds role and scope to header row children', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-table>
+        <va-table-row slot="headers">
+          <span>One</span>
+          <span>Two</span>
+          <span>Three</span>
+        </va-table-row>
+
+        <va-table-row>
+          <span>Dog</span>
+          <span>Cat</span>
+          <span>Mouse</span>
+        </va-table-row>
+      </va-table>
+    `);
+
+    const element = await page.find('va-table > va-table-row');
+    const cols = await element.findAll('span');
+    expect(element.getAttribute('role')).toEqual('row');
```

Comment:
This is kind of snuck in here. I didn't create a separate e2e test file just for `<va-table-row>` - let me know what your opinions are. Should this be its own test in this file, should it get a new file, or should I leave it as is?

Reply:
I think it's fine as is since this logic is handled in `va-table` and `va-table-row`'s only responsibility is to accept children.